packages/cli/test/acceptance/smoke.test.ts

```diff
@@ -52,7 +52,6 @@ describeOrSkip('@acceptance smoke tests', () => {
     const cmd = await run('plugins --core')
     expect(cmd.stdout).to.contain('@oclif/plugin-commands')
     expect(cmd.stdout).to.contain('@oclif/plugin-help')
```

Comment:
Will these first two items actually be included in the output? They are both now part of oclif/core.